seacrowd/sea_datasets/ac_iquad/ac_iquad.py

```diff
@@ -0,0 +1,236 @@
+    journal = "Language Resources and Evaluation",
+    issn = "1574-020X",
+    publisher = "Springer Netherlands",
+}
+"""
+
+_DATASETNAME = "ac_iquad"
+
+_DESCRIPTION = """
+This is an automatically-produced question answering dataset \
+    generated from Indonesian Wikipedia articles. Each entry \
+    in the dataset consists of a context paragraph, the \
+    question and answer, and the question's equivalent SPARQL \
+    query. Questions are separated into two subsets: simple \
+    (question consists of a single SPARQL triple pattern) and \
+    complex (question consists of two triples plus an optional \
+    typing triple).
+"""
+
+_HOMEPAGE = "https://www.kaggle.com/datasets/realdeo/indonesian-qa-generated-by-kg"
+
+_LANGUAGES = ["ind"]
+
+_LICENSE = Licenses.UNKNOWN.value
+
+_LOCAL = False
+
+_URLS = {
+    _DATASETNAME: "https://github.com/muhammadravi251001/ac-iquad/raw/main/data/ac_iquad.zip",
+}
+
+_SUPPORTED_TASKS = [Tasks.QUESTION_ANSWERING]
+
+_SOURCE_VERSION = "1.0.0"
+
+_SEACROWD_VERSION = "1.0.0"
+
+
+class ACIQuADDataset(datasets.GeneratorBasedBuilder):
+    """
+    This is an automatically-produced question answering dataset \
+    generated from Indonesian Wikipedia articles. Each entry \
+    in the dataset consists of a context paragraph, the \
+    question and answer, and the question's equivalent SPARQL \
+    query. Questions are separated into two subsets: simple \
+    (question consists of a single SPARQL triple pattern) and \
+    complex (question consists of two triples plus an optional \
+    typing triple).
+    """
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+    SEACROWD_SCHEMA_NAME = "qa"
+
+    BUILDER_CONFIGS = [
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_simple_source",
+            version=SOURCE_VERSION,
+            description=f"{_DATASETNAME} source schema",
+            schema="source",
+            subset_id=f"{_DATASETNAME}_simple",
+        ),
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_simple_seacrowd_{SEACROWD_SCHEMA_NAME}",
+            version=SEACROWD_VERSION,
+            description=f"{_DATASETNAME} SEACrowd schema",
+            schema=f"seacrowd_{SEACROWD_SCHEMA_NAME}",
+            subset_id=f"{_DATASETNAME}_simple",
+        ),
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_complex_source",
+            version=SOURCE_VERSION,
+            description=f"{_DATASETNAME} source schema",
+            schema="source",
+            subset_id=f"{_DATASETNAME}_complex",
+        ),
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_complex_seacrowd_{SEACROWD_SCHEMA_NAME}",
+            version=SEACROWD_VERSION,
+            description=f"{_DATASETNAME} SEACrowd schema",
+            schema=f"seacrowd_{SEACROWD_SCHEMA_NAME}",
+            subset_id=f"{_DATASETNAME}_complex",
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = f"{_DATASETNAME}_source"
```

Comment:
pls remove it (since the `f"{_DATASETNAME}_source"` isn't available in list config) or change it to either one (preferably `simple` ones).

Reply:
Solved on https://github.com/SEACrowd/seacrowd-datahub/pull/641/commits/95e873c9395553a8e63db0df9374d63dfae500fd commit.